StructureDefinition/England-RequestGroup.xml

```diff
@@ -27,26 +27,6 @@
   <baseDefinition value="http://hl7.org/fhir/StructureDefinition/RequestGroup" />
   <derivation value="constraint" />
   <differential>
-    <element id="RequestGroup.extension:pendingCancellation">
-      <path value="RequestGroup.extension" />
-      <sliceName value="pendingCancellation" />
-      <short value="An extension use to record if a prescription or any of its line items is pending cancellation." />
-      <type>
-        <code value="Extension" />
-        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-PendingCancellation" />
-      </type>
-      <isModifier value="false" />
-    </element>
-    <element id="RequestGroup.extension:prescriptionTaskStatusReason">
```

Comment:
Wondering why this extension was removed, we need to record the status of a prescription on the RequestGroup, and status of line items on MedicationRequests, as in the RequestGroup example